charts/builder/templates/builder-deployment.yaml

```diff
@@ -94,6 +94,10 @@ spec:
                 secretKeyRef:
                   name: builder-key-auth
                   key: builder-key
+{{- if (.Values.builder_pod_node_selector) }}
```

Comment:
Curious if there was a reason to go this route of `if` logic rather than something like:
```
- name: BUILDER_POD_NODE_SELECTOR
  value: {{ default "" .Values.builder_pod_node_selector}}
```
(would then be `BUILDER_POD_NODE_SELECTOR=` in builder env; perhaps not wanted)